zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/cache/process/CachedProcessEntity.java

```diff
@@ -9,4 +9,10 @@
 
 import java.util.List;
 
-public record CachedProcessEntity(String name, String versionTag, List<String> callElementIds) {}
+public record CachedProcessEntity(
+    String name, String versionTag, String version, List<String> callElementIds) {
+  public CachedProcessEntity(
+      final String name, final String versionTag, final List<String> callElementIds) {
+    this(name, versionTag, null, callElementIds);
+  }
+}
```

Comment:
isn't this the wrong way around? i.e. a cached entity should _always_ have a version but only sometimes a version tag. So the constructor should take a version rather than a tag, and then pass the second argument as null rather than the third. Like this:

```suggestion
public record CachedProcessEntity(
    String name, String versionTag, String version, List<String> callElementIds) {
  public CachedProcessEntity(
      final String name, final String version, final List<String> callElementIds) {
    this(name, null, version, callElementIds);
  }
}
```

Reply:
no this is the correct way round, with my solution all the current instances of `CachedProcessEntity` which use the `(name,versionTag,callElements)` constructor are unchanged, but for incidents which may require the `version` field it is filled in.